frontend/fingertips-frontend/app/results/page.tsx

```diff
@@ -96,8 +104,8 @@ export default async function Page(
         initialIndicatorSelectionState={initialState}
         searchResults={searchResults}
         availableAreaTypes={sortedByLevelAreaTypes}
-        availableAreas={availableAreas}
-        availableGroupTypes={availableGroupTypes}
+        availableAreas={sortedAlphabeticallyAvailableAreas}
```

Comment:
As discussed to add test for this. But to do this after https://github.com/dhsc-govuk/FingertipsNext/pull/206 is merged in.

This sorting functionality needs to go into the new `getAreaFilterData` function which should make the testing of this bit easier too.